dashboard/test/specs/accounts.js

```diff
@@ -31,13 +31,41 @@ describe('accounts', () => {
 
     it('can create a new account', () => {
       const alias = 'test-account-' + uuid.v4()
+      browser.waitForVisible('.ItemList button')
+      browser.scroll('.ItemList button')
       browser.click('.ItemList button')
       browser.setValue('input[name=alias]', alias)
+      browser.scroll('.FormContainer input[type=radio][value=generate]')
       browser.click('.FormContainer input[type=radio][value=generate]')
+      browser.scroll('.FormContainer button')
       browser.click('.FormContainer button')
       browser.waitForVisible('.AccountShow')
       browser.getText('.AccountShow').should.contain('Created account. Create another?')
       browser.getText('.AccountShow').should.contain(alias)
     })
   })
+
+  describe('creating receivers', () => {
+    let account
+
+    before(() => expect(testHelpers.ensureConfigured()).to.be.fulfilled
+      .then(() => {
+        return testHelpers.createAccount('receiver-test')
```

Comment:
You use implicit return at the end of this `then` chain; let's stick with that for one liners.